adapters/tlsio_openssl.c

```diff
@@ -2238,6 +2271,7 @@ CONCRETE_IO_HANDLE tlsio_openssl_create(void* io_create_parameters)
 
                     result->tls_version = OPTION_TLS_VERSION_1_0;
                     result->disable_crl_check = false;
+                    result->continue_on_crl_download_failure = false;
                     result->disable_default_verify_paths = false;
```

Comment:
Nitpick: Would it be easier to do instead do ```memset(&result, 0, sizeof(result))``` here, and then set any non-zero defaults (e.g. for tls_version)?